### ISSUE TITLE
CMS: Action Card & Action Group

### ISSUE THREAD

**Issue by prd409-utsa** [utsa-asc/college-dls#457]
This is 99% there but cannot figure out what needs to be adjusted to get it to work as expected.  

**DLS Component:** https://utsa-asc.github.io/college-dls/components/detail/action-group.html

**Formats in Cascade -** 
**actionCard:** https://walledev.it.utsa.edu/entity/open.act?id=d528840381736a1b48bb59f16e057558&type=format
**actionCardGroup:** https://walledev.it.utsa.edu/entity/open.act?id=f233509d81736a1b40f82c5437e78a4e&type=format

**Data Definition in Cascade -**
**Action-Cad-Group:** https://walledev.it.utsa.edu/entity/open.act?id=d519fb5581736a1b48bb59f16a4ff66b&type=structureddatadefinition

**Asset Factory in Cascade -**
 **Action Card Group:** https://walledev.it.utsa.edu/entity/open.act?id=d51ff5bf81736a1b48bb59f1f5467c1f&type=assetfactory

**Comment by lalov1** [comment]
@prd409-utsa I found an error in the declaration for the $cards variable:
#set($cards = $block.getStructureDataNodes("item"))

You were missing the "d" in "Structured", so I changed it to:
#set($cards = $block.getStructuredDataNodes("item"))

Not sure why it didn't error out. 

Also, the $row declaration was set to 1, but it was 0 since the count is indexed based:
#set($row = $currentPage.getStructuredDataNodes("row")[0])

Do you want to continue the work now that I've got items to return? 
https://walledev.it.utsa.edu/entity/open.act?id=d1f632af81736a1b3be11e0671f220d2&type=page

It looks like the HTML doesn't match up to the code in the DLS and there was some hard-coded styling too. Was this temporary as you were getting things to work? I could continue to work on this if you have KLESSE items to do.

**Comment by prd409-utsa** [comment]
If you can work on it that would be great!  Thanks for taking time to explain what you fixed thought that helps me learn.

**Comment by lalov1** [comment]
@prd409-utsa @garza-utsa @Uri-E Would you all review when you get a minute? 
https://walledev.it.utsa.edu/entity/open.act?id=d1f632af81736a1b3be11e0671f220d2&type=page

There were some variables for the button color and the clip-path, so I wasn't sure if a user should be able to change things like that. 

@Uri-E Would you be able to create some images with some dummy dimensions, so I could add them to the asset factory? If they have the same dimensions as another component, let me know.

**Comment by garza-utsa** [comment]
these look great, if we can maybe update the documentation to put emphasis on making sure each box is consistent (image, blurb, link), otherwise an action group may look unbalanced.

not sure if we want to make these different data parts non-optional, I'm open to placing more constraints in order to ensure a consistent visual look/feel

**Comment by garza-utsa** [comment]
follow up question for everyone, should the image clip path be on the top right here for the group, or the bottom left?

**Comment by lalov1** [comment]
In the DLS it looks like it doesn't have a clip-path, so I could remove it if everyone is okay with that:
https://utsa-asc.github.io/college-dls/components/detail/action-group.html

**Comment by Uri-E** [comment]
Give this placeholder a shot. It's about as small as it can get and still work in most screen sizes.
![utsa-cdls-action-card-460x210](https://user-images.githubusercontent.com/105820950/189435503-d35599aa-4941-4d90-b1b0-b100bf863773.jpg)

**Comment by prd409-utsa** [comment]
Sorry I missed all this yesterday!  That CEID video automatically started playing every time I saved or refreshed the page. So after being startled by it about half a dozen times, I turned down the volume on my computer. Which meant no alerts for updates here...or in Teams either. (I missed a few msg alerts there as well). Thankfully I'm through with that page. for now. 

Volume is back up to full throttle!
